src/coredump/coredump.c

```diff
@@ -1212,8 +1223,23 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                process_name  = filename_escape(context.meta[META_COMM]);
+                if (!process_name) {
+                        log_error_errno(r, "Failed to escape process name. Ignoring coredump ratelimit %m");
```

Comment:
r does not actually contain any error here.

Reply:
Thank you for your comments. I have addressed the review comments. Please check once